cartographer/mapping/trajectory_connectivity_state.cc

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cartographer/mapping/trajectory_connectivity_state.h"
+
+namespace cartographer {
+namespace mapping {
+
+TrajectoryConnectivityState::TrajectoryConnectivityState()
+    : connected_components_(),
+      last_connection_time_map_() {}
+
+void TrajectoryConnectivityState::Add(int trajectory_id) {
```

Comment:
`const int`. Similar for other arguments in this file that are never changed.

Reply:
Done